code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -942,7 +942,7 @@ Pass a positive integer as an argument to override a bot's default speed.
 				ejectpai(usr)
 
 /mob/living/simple_animal/bot/update_icon_state()
-	icon_state = "[initial(icon_state)][get_bot_flag(bot_mode_flags, BOT_MODE_ON)]"
+	icon_state = "[isnull(base_icon_state)? initial(icon_state) : initial(base_icon_state)][get_bot_flag(bot_mode_flags, BOT_MODE_ON)]"
```

Comment:
```suggestion
	icon_state = "[isnull(base_icon_state )? initial(icon_state) : base_icon_state][get_bot_flag(bot_mode_flags, BOT_MODE_ON)]"
```
the point of base icon state is that its the default icon we build off. why are you using initial with it?

Reply:
You right

Reply:
did you place the space in the wrong place